docs/sources/installation/mac.md

```diff
@@ -1,116 +1,315 @@
+Docker container using standard localhost addressing such as `localhost:8000` or
+`0.0.0.0:8376`.
 
-2. Run the installer by double-clicking the downloaded package, which will install a
-VirtualBox VM, Docker itself, and the Boot2Docker management tool.
-   ![](/installation/images/osx-installer.png)
+![Linux Architecture Diagram](/installation/images/linux_docker_host.png)
 
-3. Locate the `Boot2Docker` app in your `Applications` folder and run it.
-   Or, you can initialize Boot2Docker from the command line by running:
+In an OS X installation, the `docker` daemon is running inside a Linux virtual
+machine provided by Boot2Docker.
 
-	     $ boot2docker init
-	     $ boot2docker start
-	     $ $(boot2docker shellinit)
+![OSX Architecture Diagram](/installation/images/mac_docker_host.png)
 
-A terminal window will open and you'll see the virtual machine starting up. 
-Once you have an initialized virtual machine, you can control it with `boot2docker stop`
-and `boot2docker start`.
+In OS X, the Docker host address is the address of the Linux VM.
+When you start the `boot2docker` process, the VM is assigned an IP address. Under
+`boot2docker` ports on a container map to ports on the VM. To see this in
+practice, work through the exercises on this page.
 
-> **Note:**
-> If you see a message in the terminal that looks something like this:
->
->    `To connect the Docker client to the Docker daemon, please set: export 
-DOCKER_HOST=tcp://192.168.59.103:2375`
-> 
-you can safely set the environment variable as instructed.
 
-View the
-[Boot2Docker ReadMe](https://github.com/boot2docker/boot2docker/blob/master/README.md)
-for more information.
+## Install Boot2Docker
+ 
+1. Go to the [boo2docker/osx-installer ](
+https://github.com/boot2docker/osx-installer/releases/latest) release page.
 
-## Upgrading
+4. Download Boot2Docker by clicking `Boot2Docker-x.x.x.pkg` in the "Downloads"
+section.
 
-1. Download the latest release of the [Docker for OS X Installer](
-   https://github.com/boot2docker/osx-installer/releases/latest)
+3. Install Boot2Docker by double-clicking the package.
 
-2. If Boot2Docker is currently running, stop it with `boot2docker stop`. Then, run
-the installer package, which will update Docker and the Boot2Docker management tool.
+	The installer places Boot2Docker in your "Applications" folder.
 
-3. To complete the upgrade, you also need to update your existing virtual machine. Open a
-terminal window and run:
+The installation places the `docker` and `boot2docker` binaries in your
+`/usr/local/bin` directory.
 
-        $ boot2docker stop
-        $ boot2docker download
-        $ boot2docker start
 
-This will download an .iso image containing a fresh VM and start it up. Your upgrade is
-complete. You can test it by following the directions below.
+## Start the Boot2Docker Application
 
-## Running Docker
+To run a Docker container, you first start the `boot2docker` VM and then issue
```

Comment:
we try to keep `boot2docker` for the management tool, and use Boot2Docker as the VM and suite name - as its horribly confuseling